src/GraphQL/GraphQL.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
     <PackageReference Include="System.Memory" Version="4.5.5" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
🤔 1.1.1 ? Maybe 7.0.0 ? They all netstandard2.0 compatible.

Reply:
7.0.0
![изображение](https://user-images.githubusercontent.com/21261007/210017989-969d0d4d-d5a0-4832-b2c8-1d19933a6efc.png)


1.1.1
![изображение](https://user-images.githubusercontent.com/21261007/210018009-1d2ac31e-9c65-42b3-a594-517d88c76d47.png)

Reply:
A couple thoughts on this:
1. I believe we are only using ** interfaces ** from this package; no code.  So it really makes no difference to us which version is referenced.
2. This reference is practically only required for .NET Framework or .NET Core 2.1 targets.  Version 1.1.1 is the last 1.x release, and 1.x would be in use for projects created in that era.  It's also the second-most popular (by downloads), where 1.0.0 is most popular.
3. Anyone can reference 7.0.0 or any newer version if they require it for their code.  Again, we just need the interfaces referenced.

Reply:
I understand.

>  So it really makes no difference to us which version is referenced. Anyone can reference 7.0.0 or any newer version if they require it for their code.

True. I just balance between the desire to use the minimum version as a library developer and maximum as its consumer, so as not to add packages on my own.

Reply:
When it matters, though, it really bites hard.  For instance, @SimonCropp keeps his libraries up to date with the latest versions of all dependencies always.  Most recently, MS released a **_completely_** broken copy of Microsoft.SqlClient (v5.0.1), which can crash servers within hours.  Since his LocalDb package references 5.0.1, my application pulls in the buggy version of SqlClient.  Fortunately it's only used in tests and does not matter there, but in a similar vein, the latest version of his GraphQL.EntityFramework package can only be used EF 7.0.1, even though the interfaces it uses are probably compatible as far back as EF Core 2 or 3.  I have a couple applications that run on .NET Framework, and so I'm stuck on EF Core 3 for certain server projects, precluding me from using GraphQL.EntityFramework, or any recent version anyway.

So my 'balance' here is an attempt to reference the same major version as would be done by other code for the same target framework.  For example, for a `net5.0` TFM, reference version `5.0.x` of `Microsoft.Extensions.Options`.  This way, the major frameworks likely to be used by the client application (ASP.NET Core, for example) are likely to be using the same major version of side packages as you are.  In this case, I felt that version 1.1.1 would apply the best.

Even better is to have no dependencies, which we have achieved nicely so far, but in this case we need one for the .NET Standard 2.0 TFM.

Reply:
I completely understand and agree. I reason in the same way.

>  SimonCropp keeps his libraries up to date with the latest versions of all dependencies always. 

Yeah. I know :) I'm familiar with such an extreme and do not encourage that strategy.

Regarding `Bcl` package I just thought that it would not "bite". It looks rather "harmless". Nevertheless, I'm fine with `1.1.1`.

Reply:
😞 3 AM here but still did not review PRs about subscriptions / AOT, I'm done for today. Very-very-very good work in conventions repo.